docs/resources/icon-library.md

```diff
@@ -1,9 +1,10 @@
 ---
-title: Icons
+title: Icon library
+see_also:
+  - title: Icon usage
+    href: /usage/icons/
 ---
 
-See the [Icons code page](./code/) for usage information.
-
 <div class="flex flex-wrap">
   {% for icon in icons -%}
     <div class="w-1/3 mb-80">
```

Comment:
I'm assuming we should also remove the `<a>` below; I liked Corey's idea for a "copy name/id to clipboard" feature, as a potential for future interactivity on this page